solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -232,7 +232,8 @@ contract RandomBeacon is Ownable {
 
     event DkgResultApproved(
         bytes32 indexed resultHash,
-        address indexed approver
+        address indexed approver,
+        bytes32 membersHash
```

Comment:
Why do we need `membersHash` in this event?

Reply:
Added to verify members against `DkgResultSubmitted`, but you're right, we have `resultHash` which include members. Looks like it will require a bit more lines of code to verify dkg members for contract callers if needed, but we save gas on not hashing members and not emitting them. 075d32c